packages/components/Menu/src/MenuItem/MenuItem.styling.ts

```diff
@@ -52,11 +52,7 @@ export const stylingSettings: UseStylingOptions<MenuItemProps, MenuItemSlotProps
           color: tokens.submenuIndicatorColor,
           height: tokens.submenuIndicatorSize,
           width: tokens.submenuIndicatorSize,
-          viewBox:
-            '0 0 ' +
-            (tokens.submenuIndicatorSize - tokens.submenuIndicatorPadding * 2) +
-            ' ' +
-            (tokens.submenuIndicatorSize - tokens.submenuIndicatorPadding * 2),
+          viewBox: '0 0 ' + tokens.submenuIndicatorSize + ' ' + tokens.submenuIndicatorSize,
```

Comment:
Any reason why we're not using [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#string_interpolation) here?

Reply:
Thanks for the suggestion! fixed in 56ec502348e57cf01aaef0249993cb3444cc752a